src/index.html

```diff
@@ -16,15 +16,15 @@ <h1 id="page-title">Leaderboard</h1>
         <h3 id="score-title">Recent Scores</h3>
         <button id="refresh-button" type="button">Refresh</button>
       </header>
-      <ul id="score-list">
-        <li class="list-item">Name: 180</li>
+      <ul id="score-list" class="flex">
+        <!-- <li class="list-item">Name: 180</li>
         <li class="list-item">Name: 120</li>
         <li class="list-item">Name: 110</li>
         <li class="list-item">Name: 100</li>
         <li class="list-item">Name: 80</li>
         <li class="list-item">Name: 75</li>
         <li class="list-item">Name: 50</li>
-        <li class="list-item">Name: 10</li>
+        <li class="list-item">Name: 10</li> -->
```

Comment:
- [ ] [optional] It would be nice if you remove commented codes

Reply:
Thank you, I'll get rid of commented code.